core/src/test/java/org/infinispan/filter/CompositeKeyValueFilterConverter.java

```diff
@@ -14,6 +19,7 @@
  * @author wburns
  * @since 7.0
  */
+@SerializeWith(CompositeKeyValueFilterConverter.Ext.class)
```

Comment:
> Also, shouldn't the externalizer class be called Externalizer?

Are we in the mood of nitpicking? ;)

Since you are asking, having named the externalizer `Externalizer`, I would have had to use the fully qualified name in `implements org.infinispan.commons.marshall.Externalizer`. So instead of doing that, I simply called it `Ext` which is the prefix of `Externalizer`.

Hope that clears up this confusion ;)